recipes/radical.utils/LICENSE.md

```diff
@@ -0,0 +1,207 @@
+radical.utils is licensed under the MIT License (MIT)
```

Comment:
You don't need this file, if it's in the source tarball.

Reply:
Thank you!